packages/venia-concept/src/RootComponents/Product/Product.js

```diff
@@ -172,4 +129,11 @@ class Product extends Component {
     }
 }
 
-export default classify(defaultClasses)(Product);
+export default connect(
+    ({ cart: { guestCartId } = {} }) => ({ guestCartId }),
```

Comment:
I personally find this difficult to visually parse without sitting and concentrating. Can we maybe try something less terse?

Reply:
Agreed. Nested destructuring is tough to read.